core/src/main/java/nu/yona/server/analysis/service/IntervalActivityDTO.java

```diff
@@ -58,13 +64,31 @@ public UUID getGoalID()
 		return goalID;
 	}
 
+	@JsonInclude(Include.NON_NULL)
 	public List<Integer> getSpread()
 	{
 		return spread;
 	}
 
-	public int getTotalActivityDurationMinutes()
+	@JsonInclude(Include.NON_NULL)
```

Comment:
Change this into ``Include.NON_EMPTY``. A collection that is null is against good coding practices.